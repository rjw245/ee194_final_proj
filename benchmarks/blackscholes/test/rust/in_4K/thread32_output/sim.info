{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread32_output',
             '-n',
             '32',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../rust/bs',
             '32',
             '../../inputs/in_4K.txt'],
 'git_revision': '',
 'host': 'dell26',
 'pin_version': 71313,
 'rusage': (891.484473,
            4.346339,
            319352,
            0,
            0,
            0,
            485321,
            2,
            0,
            488,
            16720,
            0,
            0,
            0,
            285203,
            138759),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=32 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/rust/thread32_output --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../rust/bs 32 ../../inputs/in_4K.txt'],
 't_elapsed': 1365.325945854187,
 't_start': 1460948560.178553,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 6925680640L}
