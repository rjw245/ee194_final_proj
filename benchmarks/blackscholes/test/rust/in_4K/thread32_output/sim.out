                                     | Core 0     | Core 1     | Core 2     | Core 3     | Core 4     | Core 5     | Core 6     | Core 7     | Core 8     | Core 9     | Core 10    | Core 11    | Core 12    | Core 13    | Core 14    | Core 15    | Core 16    | Core 17    | Core 18    | Core 19    | Core 20    | Core 21    | Core 22    | Core 23    | Core 24    | Core 25    | Core 26    | Core 27    | Core 28    | Core 29    | Core 30    | Core 31   
  Instructions                       |  276747331 |    8392084 |     136718 |      65052 |      68728 |      69212 |      69195 |      66604 |      68591 |      69734 |      67675 |      69895 |      70825 |      70253 |      69542 |      71327 |      72145 |      73859 |      67726 |      68350 |      72557 |      69607 |      69625 |      68015 |      66707 |      58144 |      69852 |      69177 |      70414 |      58204 |      62502 |      72210
  Cycles                             |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040 |  166899040
  IPC                                |       1.66 |       0.05 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00 |       0.00
  Time (ns)                          |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000 |   62744000
  Idle time (ns)                     |    2497920 |   60184178 |   62697879 |   62720386 |   62707144 |   62718242 |   62720179 |   62721241 |   62704990 |   62714039 |   62720747 |   62717924 |   62704830 |   62711205 |   62718127 |   62717996 |   62709397 |   62716717 |   62703832 |   62719043 |   62715384 |   62711284 |   62709012 |   62721355 |   62713623 |   62705727 |   62719558 |   62717687 |   62714033 |   62706130 |   62715462 |   62718384
  Idle time (%)                      |       4.0% |      95.9% |      99.9% |     100.0% |      99.9% |     100.0% |     100.0% |     100.0% |      99.9% |     100.0% |     100.0% |     100.0% |      99.9% |      99.9% |     100.0% |     100.0% |      99.9% |     100.0% |      99.9% |     100.0% |     100.0% |      99.9% |      99.9% |     100.0% |     100.0% |      99.9% |     100.0% |     100.0% |     100.0% |      99.9% |     100.0% |     100.0%
Branch predictor stats               |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
  num correct                        |    9816831 |     303391 |      13215 |       6023 |       6429 |       6605 |       6604 |       6333 |       6625 |       6742 |       6516 |       6865 |       6757 |       6676 |       6803 |       7045 |       7115 |       7391 |       6522 |       6640 |       7130 |       6758 |       6607 |       6573 |       6392 |       5382 |       6801 |       6714 |       6868 |       5397 |       5851 |       7004
  num incorrect                      |     429920 |       6200 |        962 |        611 |        737 |        673 |        673 |        600 |        645 |        659 |        613 |        655 |        648 |        617 |        654 |        677 |        698 |        705 |        615 |        622 |        711 |        656 |        620 |        618 |        602 |        499 |        655 |        642 |        667 |        499 |        509 |        632
  misprediction rate                 |      4.20% |      2.00% |      6.79% |      9.21% |     10.28% |      9.25% |      9.25% |      8.65% |      8.87% |      8.90% |      8.60% |      8.71% |      8.75% |      8.46% |      8.77% |      8.77% |      8.93% |      8.71% |      8.62% |      8.57% |      9.07% |      8.85% |      8.58% |      8.59% |      8.61% |      8.48% |      8.78% |      8.73% |      8.85% |      8.46% |      8.00% |      8.28%
  mpki                               |       1.55 |       0.74 |       7.04 |       9.39 |      10.72 |       9.72 |       9.73 |       9.01 |       9.40 |       9.45 |       9.06 |       9.37 |       9.15 |       8.78 |       9.40 |       9.49 |       9.67 |       9.55 |       9.08 |       9.10 |       9.80 |       9.42 |       8.90 |       9.09 |       9.02 |       8.58 |       9.38 |       9.28 |       9.47 |       8.57 |       8.14 |       8.75
TLB Summary                          |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
  I-TLB                              |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num accesses                     |   56365830 |    1412191 |      18713 |       8775 |       9174 |       9536 |       9518 |       9058 |       9458 |       9641 |       9265 |       9693 |       9819 |       9697 |       9615 |       9928 |      10057 |      10368 |       9283 |       9383 |      10091 |       9573 |       9578 |       9286 |       9066 |       7548 |       9611 |       9494 |       9693 |       7556 |       8304 |      10009
    num misses                       |        194 |         81 |         71 |         68 |         73 |         68 |         68 |         68 |         68 |         68 |         68 |         68 |         69 |         68 |         68 |         68 |         68 |         68 |         68 |         68 |         68 |         68 |         68 |         68 |         68 |         59 |         68 |         68 |         68 |         59 |         60 |         68
    miss rate                        |      0.00% |      0.01% |      0.38% |      0.77% |      0.80% |      0.71% |      0.71% |      0.75% |      0.72% |      0.71% |      0.73% |      0.70% |      0.70% |      0.70% |      0.71% |      0.68% |      0.68% |      0.66% |      0.73% |      0.72% |      0.67% |      0.71% |      0.71% |      0.73% |      0.75% |      0.78% |      0.71% |      0.72% |      0.70% |      0.78% |      0.72% |      0.68%
    mpki                             |       0.00 |       0.01 |       0.52 |       1.05 |       1.06 |       0.98 |       0.98 |       1.02 |       0.99 |       0.98 |       1.00 |       0.97 |       0.97 |       0.97 |       0.98 |       0.95 |       0.94 |       0.92 |       1.00 |       0.99 |       0.94 |       0.98 |       0.98 |       1.00 |       1.02 |       1.01 |       0.97 |       0.98 |       0.97 |       1.01 |       0.96 |       0.94
  D-TLB                              |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num accesses                     |  184150916 |    4810249 |      48339 |      22884 |      24372 |      23804 |      23824 |      23087 |      23444 |      23825 |      23211 |      23619 |      25261 |      25197 |      23522 |      23899 |      24208 |      24547 |      23225 |      23305 |      24477 |      23715 |      25040 |      23287 |      23034 |      19677 |      23723 |      23634 |      23927 |      19686 |      21935 |      25509
    num misses                       |      16340 |        192 |        158 |         64 |        104 |         70 |         69 |         68 |         74 |         73 |         76 |         88 |         82 |         88 |         80 |         82 |         88 |         82 |         78 |         87 |         90 |         89 |        101 |         92 |         90 |         80 |         95 |         96 |        101 |         85 |         95 |        102
    miss rate                        |      0.01% |      0.00% |      0.33% |      0.28% |      0.43% |      0.29% |      0.29% |      0.29% |      0.32% |      0.31% |      0.33% |      0.37% |      0.32% |      0.35% |      0.34% |      0.34% |      0.36% |      0.33% |      0.34% |      0.37% |      0.37% |      0.38% |      0.40% |      0.40% |      0.39% |      0.41% |      0.40% |      0.41% |      0.42% |      0.43% |      0.43% |      0.40%
    mpki                             |       0.06 |       0.02 |       1.16 |       0.98 |       1.51 |       1.01 |       1.00 |       1.02 |       1.08 |       1.05 |       1.12 |       1.26 |       1.16 |       1.25 |       1.15 |       1.15 |       1.22 |       1.11 |       1.15 |       1.27 |       1.24 |       1.28 |       1.45 |       1.35 |       1.35 |       1.38 |       1.36 |       1.39 |       1.43 |       1.46 |       1.52 |       1.41
  L2 TLB                             |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num accesses                     |      16534 |        273 |        229 |        132 |        177 |        138 |        137 |        136 |        142 |        141 |        144 |        156 |        151 |        156 |        148 |        150 |        156 |        150 |        146 |        155 |        158 |        157 |        169 |        160 |        158 |        139 |        163 |        164 |        169 |        144 |        155 |        170
    num misses                       |        519 |        215 |        179 |        125 |        160 |        130 |        130 |        129 |        132 |        132 |        135 |        143 |        142 |        142 |        139 |        141 |        144 |        142 |        135 |        146 |        147 |        145 |        151 |        148 |        147 |        132 |        150 |        150 |        153 |        135 |        142 |        159
    miss rate                        |      3.14% |     78.75% |     78.17% |     94.70% |     90.40% |     94.20% |     94.89% |     94.85% |     92.96% |     93.62% |     93.75% |     91.67% |     94.04% |     91.03% |     93.92% |     94.00% |     92.31% |     94.67% |     92.47% |     94.19% |     93.04% |     92.36% |     89.35% |     92.50% |     93.04% |     94.96% |     92.02% |     91.46% |     90.53% |     93.75% |     91.61% |     93.53%
    mpki                             |       0.00 |       0.03 |       1.31 |       1.92 |       2.33 |       1.88 |       1.88 |       1.94 |       1.92 |       1.89 |       1.99 |       2.05 |       2.00 |       2.02 |       2.00 |       1.98 |       2.00 |       1.92 |       1.99 |       2.14 |       2.03 |       2.08 |       2.17 |       2.18 |       2.20 |       2.27 |       2.15 |       2.17 |       2.17 |       2.32 |       2.27 |       2.20
Cache Summary                        |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
  Cache L1-I                         |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num cache accesses               |   56365830 |    1412191 |      18713 |       8775 |       9174 |       9536 |       9518 |       9058 |       9458 |       9641 |       9265 |       9693 |       9819 |       9697 |       9615 |       9928 |      10057 |      10368 |       9283 |       9383 |      10091 |       9573 |       9578 |       9286 |       9066 |       7548 |       9611 |       9494 |       9693 |       7556 |       8304 |      10009
    num cache misses                 |    1292095 |       1097 |       1647 |        885 |        935 |        903 |        908 |        890 |        875 |        888 |        885 |        884 |        893 |        886 |        886 |        877 |        908 |        911 |        881 |        887 |        919 |        895 |        880 |        887 |        880 |        703 |        902 |        906 |        905 |        703 |        719 |        888
    miss rate                        |      2.29% |      0.08% |      8.80% |     10.09% |     10.19% |      9.47% |      9.54% |      9.83% |      9.25% |      9.21% |      9.55% |      9.12% |      9.09% |      9.14% |      9.21% |      8.83% |      9.03% |      8.79% |      9.49% |      9.45% |      9.11% |      9.35% |      9.19% |      9.55% |      9.71% |      9.31% |      9.39% |      9.54% |      9.34% |      9.30% |      8.66% |      8.87%
    mpki                             |       4.67 |       0.13 |      12.05 |      13.60 |      13.60 |      13.05 |      13.12 |      13.36 |      12.76 |      12.73 |      13.08 |      12.65 |      12.61 |      12.61 |      12.74 |      12.30 |      12.59 |      12.33 |      13.01 |      12.98 |      12.67 |      12.86 |      12.64 |      13.04 |      13.19 |      12.09 |      12.91 |      13.10 |      12.85 |      12.08 |      11.50 |      12.30
  Cache L1-D                         |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num cache accesses               |  184152074 |    4810512 |      49429 |      23292 |      24750 |      24431 |      24438 |      23636 |      24134 |      24514 |      23853 |      24429 |      25910 |      25805 |      24285 |      24738 |      25060 |      25488 |      23874 |      24008 |      25270 |      24420 |      25630 |      23918 |      23616 |      20150 |      24416 |      24308 |      24638 |      20165 |      22510 |      26217
    num cache misses                 |      17193 |       3649 |       2010 |        707 |        799 |        741 |        705 |        698 |        728 |        689 |        667 |        756 |       1404 |       1413 |        730 |        741 |        793 |        759 |        709 |        726 |        771 |        763 |       1368 |        687 |        740 |        542 |        724 |        776 |        756 |        580 |       1220 |       1366
    miss rate                        |      0.01% |      0.08% |      4.07% |      3.04% |      3.23% |      3.03% |      2.88% |      2.95% |      3.02% |      2.81% |      2.80% |      3.09% |      5.42% |      5.48% |      3.01% |      3.00% |      3.16% |      2.98% |      2.97% |      3.02% |      3.05% |      3.12% |      5.34% |      2.87% |      3.13% |      2.69% |      2.97% |      3.19% |      3.07% |      2.88% |      5.42% |      5.21%
    mpki                             |       0.06 |       0.43 |      14.70 |      10.87 |      11.63 |      10.71 |      10.19 |      10.48 |      10.61 |       9.88 |       9.86 |      10.82 |      19.82 |      20.11 |      10.50 |      10.39 |      10.99 |      10.28 |      10.47 |      10.62 |      10.63 |      10.96 |      19.65 |      10.10 |      11.09 |       9.32 |      10.36 |      11.22 |      10.74 |       9.96 |      19.52 |      18.92
  Cache L2                           |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num cache accesses               |    1309615 |       4768 |       3724 |       1598 |       1799 |       1693 |       1659 |       1605 |       1645 |       1622 |       1584 |       1698 |       2343 |       2335 |       1649 |       1648 |       1746 |       1726 |       1632 |       1665 |       1741 |       1717 |       2288 |       1614 |       1663 |       1329 |       1672 |       1720 |       1721 |       1363 |       2014 |       2301
    num cache misses                 |      14032 |       4171 |       2560 |       1528 |       1666 |       1574 |       1538 |       1527 |       1539 |       1511 |       1495 |       1572 |       2029 |       2026 |       1552 |       1568 |       1631 |       1600 |       1532 |       1558 |       1617 |       1598 |       1979 |       1517 |       1559 |       1211 |       1565 |       1616 |       1594 |       1252 |       1802 |       1988
    miss rate                        |      1.07% |     87.48% |     68.74% |     95.62% |     92.61% |     92.97% |     92.71% |     95.14% |     93.56% |     93.16% |     94.38% |     92.58% |     86.60% |     86.77% |     94.12% |     95.15% |     93.41% |     92.70% |     93.87% |     93.57% |     92.88% |     93.07% |     86.49% |     93.99% |     93.75% |     91.12% |     93.60% |     93.95% |     92.62% |     91.86% |     89.47% |     86.40%
    mpki                             |       0.05 |       0.50 |      18.72 |      23.49 |      24.24 |      22.74 |      22.23 |      22.93 |      22.44 |      21.67 |      22.09 |      22.49 |      28.65 |      28.84 |      22.32 |      21.98 |      22.61 |      21.66 |      22.62 |      22.79 |      22.29 |      22.96 |      28.42 |      22.30 |      23.37 |      20.83 |      22.40 |      23.36 |      22.64 |      21.51 |      28.83 |      27.53
  Cache L3                           |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num cache accesses               |      14397 |       4194 |       2627 |       1534 |       1731 |       1623 |       1584 |       1544 |       1581 |       1556 |       1527 |       1630 |       2077 |       2063 |       1585 |       1598 |       1676 |       1656 |       1573 |       1610 |       1668 |       1657 |       2020 |       1557 |       1602 |       1295 |       1611 |       1654 |       1654 |       1332 |       1878 |       2036
    num cache misses                 |      12124 |        867 |       1533 |        446 |       1632 |        574 |        506 |        484 |       1215 |        707 |        539 |        583 |       1665 |       1249 |        528 |        519 |        910 |        620 |       1157 |        555 |        735 |        894 |       1402 |        485 |        896 |       1027 |        524 |        561 |        840 |        889 |       1012 |       1032
    miss rate                        |     84.21% |     20.67% |     58.36% |     29.07% |     94.28% |     35.37% |     31.94% |     31.35% |     76.85% |     45.44% |     35.30% |     35.77% |     80.16% |     60.54% |     33.31% |     32.48% |     54.30% |     37.44% |     73.55% |     34.47% |     44.06% |     53.95% |     69.41% |     31.15% |     55.93% |     79.31% |     32.53% |     33.92% |     50.79% |     66.74% |     53.89% |     50.69%
    mpki                             |       0.04 |       0.10 |      11.21 |       6.86 |      23.75 |       8.29 |       7.31 |       7.27 |      17.71 |      10.14 |       7.96 |       8.34 |      23.51 |      17.78 |       7.59 |       7.28 |      12.61 |       8.39 |      17.08 |       8.12 |      10.13 |      12.84 |      20.14 |       7.13 |      13.43 |      17.66 |       7.50 |       8.11 |      11.93 |      15.27 |      16.19 |      14.29
DRAM summary                         |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
  num dram accesses                  |       4238 |          0 |          0 |          0 |       4204 |          0 |          0 |          0 |       4313 |          0 |          0 |          0 |       4214 |          0 |          0 |          0 |       4120 |          0 |          0 |          0 |       3987 |          0 |          0 |          0 |       4013 |          0 |          0 |          0 |       4032 |          0 |          0 |          0
  average dram access latency (ns)   |      55.62 |        inf |        inf |        inf |      56.01 |        inf |        inf |        inf |      55.69 |        inf |        inf |        inf |      56.18 |        inf |        inf |        inf |      56.05 |        inf |        inf |        inf |      55.65 |        inf |        inf |        inf |      55.56 |        inf |        inf |        inf |      55.39 |        inf |        inf |        inf
  average dram queueing delay        |       2.19 |       0.00 |       0.00 |       0.00 |       2.58 |       0.00 |       0.00 |       0.00 |       2.26 |       0.00 |       0.00 |       0.00 |       2.76 |       0.00 |       0.00 |       0.00 |       2.62 |       0.00 |       0.00 |       0.00 |       2.22 |       0.00 |       0.00 |       0.00 |       2.13 |       0.00 |       0.00 |       0.00 |       1.96 |       0.00 |       0.00 |       0.00
  average dram bandwidth utilization |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00% |      0.00%
Coherency Traffic                    |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
  num loads from dram                |       3279 |        317 |         85 |         27 |         25 |         35 |         33 |         26 |        125 |         50 |         37 |         33 |        133 |         46 |         30 |         27 |         82 |         36 |        108 |         27 |         66 |         92 |         57 |         26 |         92 |        102 |         25 |         30 |         82 |         91 |         27 |         35
  num loads from dram cache          |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  num loads from remote cache        |        188 |         46 |        103 |         14 |        309 |         72 |         65 |         36 |         73 |         63 |         51 |         92 |         78 |         72 |         55 |         57 |         65 |         93 |         76 |         79 |         82 |         89 |         68 |         64 |         74 |        145 |         69 |         63 |         84 |        145 |        145 |         72
